spec.html

```diff
@@ -7216,7 +7225,7 @@ <h1>Realms</h1>
             [[GlobalEnv]]
           </td>
           <td>
-            Lexical Environment
+            global Environment Record
```

Comment:
Capitalise?

Reply:
The spec is pretty consistent on the convention that, in the name of an ER 'subtype', the word before "Environment Record" is lower-case, only being capitalized where forced (e.g., start of sentence or clause/caption title). So the question is whether a table-cell forces initial-capital.

The spec isn't entirely clear on this question. Certainly, most table-cells start with a cap, but a lot of those would be capitalized anyway. (E.g., it's the name of an operation, or the start of a sentence.) And there are about 17 cases where a table-cell starts with a lower-case letter.

I think I'll ask for editorial direction on this.

Reply:
Can you link to some of the 17 cases?

Reply:
Sorry, was looking at the wrong version, it's 15 cases in the status quo:
- 2 in https://tc39.es/ecma262/#table-8
- 1 in https://tc39.es/ecma262/#table-22
- 1 in https://tc39.es/ecma262/#table-script-records
- 1 in https://tc39.es/ecma262/#table-38
- 2 in https://tc39.es/ecma262/#table-43
- 8 in https://tc39.es/ecma262/#sec-date-time-string-format

Reply:
I don't think a table cell should force an initial cap.

Reply:
I agree, but there's no reason for "global" to be lowercased - it's the Global Environment Record.

Reply:
Yeah I'd personally prefer to change the capitalisation for these terms.